db/models.py

```diff
@@ -50,3 +55,65 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(AUTH_USER_MODEL,
+                             on_delete=models.CASCADE,
+                             null=True,
+                             blank=True,
+                             related_name="orders")
```

Comment:
The `user` field in the `Order` model allows `null` and `blank` values. Ensure this behavior is intentional and aligns with your application's logic, as it allows orders to exist without an associated user.